specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -2232,11 +2232,11 @@
       "properties": {
         "timeAfterCreate": {
           "type": "string",
-          "description": "Time after creation to attempt rotate. It will be in ISO 8601 format. Example: 90 days : \"P90D\" "
+          "description": "Time after creation to attempt rotate. It only applies to rotate. It will be in ISO 8601 format. Example: 90 days : \"P90D\" "
         },
         "timeBeforeExpiry": {
           "type": "string",
-          "description": "Time before expiry to attempt rotate. It will be in ISO 8601 format. Example: 90 days : \"P90D\""
+          "description": "Time before expiry to attempt rotate or notify. It will be in ISO 8601 format. Example: 90 days : \"P90D\""
```

Comment:
```suggestion
          "description": "Time before expiry to attempt to rotate or notify. It will be in ISO 8601 duration format. Example: 90 days : \"P90D\""
```